requirements.txt

```diff
@@ -30,7 +30,7 @@ pep8==1.4.6 # rq.filter: ==1.4.6
 Pillow==3.3.1
 psycopg2==2.6.2                          # http://initd.org/psycopg/docs/news.html
 python-dateutil==2.5.3                   # https://github.com/dateutil/dateutil/blob/master/NEWS
-python-omgeo==2.0.0
+git+https://github.com/maurizi/python-omgeo.git@authenticate-esri-wgs
```

Comment:
TODO: Switch this to a new release once https://github.com/azavea/python-omgeo/pull/40 is merged